### ISSUE TITLE
Bug with I18n with Files collection when using dynamic path {{locale}}

### ISSUE THREAD

**Issue by AntoineBx** [sveltia/sveltia-cms#232]
Hello Kioshino! I might have a bug when i write files collection when i mix i18n: true & false for each field. More specifically when i put 
dynamic {{locale}} in the path for one field and not the other

 I'm pretty sure it used to work in 0.38 but now it crash saying : 
`Uncaught TypeError: Cannot read properties of undefined (reading 'path')`

```
...
load_config_file: false
media_folder: public/img
public_folder: /img
i18n:
  structure: multiple_folders
  locales:
    - en
    - fr
  default_locale: fr
collections:
  - label: Global Options
    name: options
    i18n: true
    editor:
      preview: false
    files:
      - label: languages
        name: langs
        i18n: false
        file: content/_data/langs.yml           //HERE
        fields:
          - label: Content
            name: content
            widget: markdown
            i18n: false
      - label: Search engine optimisation
        name: seo
        i18n: true
        file: content/_data/{{locale}}/seo.yml    //VS HERE
        fields:
          - label: Title
            name: title
            widget: string
            i18n: true
            hint: Title of the page, recommended 35-70 characters
```

**Comment by kyoshino** [comment]
Thanks for the bug report. Will check.

**Comment by kyoshino** [comment]
Fixed in [v0.42.2](https://github.com/sveltia/sveltia-cms/releases/tag/v0.42.2) 🙇🏼
